app/views/discovered_hosts/show.html.erb

```diff
@@ -1,3 +1,6 @@
+<% content_for(:stylesheets) do %>
+  <%= stylesheet 'discovery/show' %>
```

Comment:
core has a mechanism for this already, not sure that you need to write your own..
example at https://github.com/theforeman/foreman_openscap/blob/94eea5947265ed270b0176f6f94ca80d5923dab4/app/views/arf_reports/show.html.erb#L3

Reply:
if its done incorrectly there, please send a PR to fix it too :)
also, not sure if you saw it, https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin has a lot of information about services that core provider for plugins

Reply:
I have read and used that page, that's how I was able to make foreman compile it for me :)